diff -Nur cdk-4.9.10-20020727.orig/include/alphalist.h cdk-4.9.10-20020727/include/alphalist.h
--- cdk-4.9.10-20020727.orig/include/alphalist.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/alphalist.h	Fri Mar 28 14:40:43 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
@@ -52,8 +52,8 @@
  * SUCH DAMAGE.
  */
 
-#include <entry.h>
-#include <scroll.h>
+#include <cdk/entry.h>
+#include <cdk/scroll.h>
 
 /*
  * Define the CDK alphalist widget structure.
diff -Nur cdk-4.9.10-20020727.orig/include/binding.h cdk-4.9.10-20020727/include/binding.h
--- cdk-4.9.10-20020727.orig/include/binding.h	Tue Apr 30 23:43:10 2002
+++ cdk-4.9.10-20020727/include/binding.h	Fri Mar 28 14:40:48 2003
@@ -12,7 +12,7 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
 #endif
 
diff -Nur cdk-4.9.10-20020727.orig/include/button.h cdk-4.9.10-20020727/include/button.h
--- cdk-4.9.10-20020727.orig/include/button.h	Thu Jul 11 00:19:03 2002
+++ cdk-4.9.10-20020727/include/button.h	Fri Mar 28 14:41:54 2003
@@ -6,15 +6,13 @@
 #ifndef CDKBUTTON_H
 #define CDKBUTTON_H	1
 
-#include "cdk.h"
-
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/buttonbox.h cdk-4.9.10-20020727/include/buttonbox.h
--- cdk-4.9.10-20020727.orig/include/buttonbox.h	Tue Jul 16 02:54:51 2002
+++ cdk-4.9.10-20020727/include/buttonbox.h	Fri Mar 28 14:42:05 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/calendar.h cdk-4.9.10-20020727/include/calendar.h
--- cdk-4.9.10-20020727.orig/include/calendar.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/calendar.h	Fri Mar 28 14:42:15 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/cdk.h cdk-4.9.10-20020727/include/cdk.h
--- cdk-4.9.10-20020727.orig/include/cdk.h	Sat Jul 20 02:20:56 2002
+++ cdk-4.9.10-20020727/include/cdk.h	Fri Mar 28 14:42:59 2003
@@ -42,8 +42,8 @@
  * SUCH DAMAGE.
  */
 
-#include <cdk_config.h>
-#include <cdk_version.h>
+#include <cdk/cdk_config.h>
+#include <cdk/cdk_version.h>
 
 #ifdef	CDK_PERL_EXT
 #undef	instr
@@ -283,45 +283,45 @@
 /*
  * These header files define miscellaneous values and prototypes.
  */
-#include <cdkscreen.h>
-#include <curdefs.h>
-#include <binding.h>
-#include <cdk_util.h>
-#include <cdk_objs.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/curdefs.h>
+#include <cdk/binding.h>
+#include <cdk/cdk_util.h>
+#include <cdk/cdk_objs.h>
 
 /*
  * Include the CDK widget header files.
  */
-#include <alphalist.h>
-#include <buttonbox.h>
-#include <calendar.h>
-#include <dialog.h>
-#include <entry.h>
-#include <fscale.h>
-#include <fselect.h>
-#include <graph.h>
-#include <histogram.h>
-#include <itemlist.h>
-#include <label.h>
-#include <marquee.h>
-#include <matrix.h>
-#include <mentry.h>
-#include <menu.h>
-#include <radio.h>
-#include <scale.h>
-#include <scroll.h>
-#include <selection.h>
-#include <slider.h>
-#include <swindow.h>
-#include <template.h>
-#include <viewer.h>
-#include <traverse.h>
-#include <button.h>
+#include <cdk/alphalist.h>
+#include <cdk/buttonbox.h>
+#include <cdk/calendar.h>
+#include <cdk/dialog.h>
+#include <cdk/entry.h>
+#include <cdk/fscale.h>
+#include <cdk/fselect.h>
+#include <cdk/graph.h>
+#include <cdk/histogram.h>
+#include <cdk/itemlist.h>
+#include <cdk/label.h>
+#include <cdk/marquee.h>
+#include <cdk/matrix.h>
+#include <cdk/mentry.h>
+#include <cdk/menu.h>
+#include <cdk/radio.h>
+#include <cdk/scale.h>
+#include <cdk/scroll.h>
+#include <cdk/selection.h>
+#include <cdk/slider.h>
+#include <cdk/swindow.h>
+#include <cdk/template.h>
+#include <cdk/viewer.h>
+#include <cdk/traverse.h>
+#include <cdk/button.h>
 
 /*
  * Low-level object drawing
  */
-#include <draw.h>
+#include <cdk/draw.h>
 
 #ifdef __cplusplus
 }
diff -Nur cdk-4.9.10-20020727.orig/include/cdk_objs.h cdk-4.9.10-20020727/include/cdk_objs.h
--- cdk-4.9.10-20020727.orig/include/cdk_objs.h	Sat Jul 27 01:33:16 2002
+++ cdk-4.9.10-20020727/include/cdk_objs.h	Fri Mar 28 14:43:11 2003
@@ -12,10 +12,10 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/cdk_util.h cdk-4.9.10-20020727/include/cdk_util.h
--- cdk-4.9.10-20020727.orig/include/cdk_util.h	Mon Jul 15 00:09:15 2002
+++ cdk-4.9.10-20020727/include/cdk_util.h	Fri Mar 28 14:43:20 2003
@@ -12,10 +12,10 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/cdkscreen.h cdk-4.9.10-20020727/include/cdkscreen.h
--- cdk-4.9.10-20020727.orig/include/cdkscreen.h	Wed Jul 17 23:06:24 2002
+++ cdk-4.9.10-20020727/include/cdkscreen.h	Fri Mar 28 14:43:27 2003
@@ -12,7 +12,7 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
 #endif
 
diff -Nur cdk-4.9.10-20020727.orig/include/curdefs.h cdk-4.9.10-20020727/include/curdefs.h
--- cdk-4.9.10-20020727.orig/include/curdefs.h	Wed May  1 00:04:15 2002
+++ cdk-4.9.10-20020727/include/curdefs.h	Fri Mar 28 14:43:38 2003
@@ -4,7 +4,7 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
 #endif
 
diff -Nur cdk-4.9.10-20020727.orig/include/dialog.h cdk-4.9.10-20020727/include/dialog.h
--- cdk-4.9.10-20020727.orig/include/dialog.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/dialog.h	Fri Mar 28 14:43:47 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/draw.h cdk-4.9.10-20020727/include/draw.h
--- cdk-4.9.10-20020727.orig/include/draw.h	Wed Jul 17 12:04:54 2002
+++ cdk-4.9.10-20020727/include/draw.h	Fri Mar 28 14:43:54 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/entry.h cdk-4.9.10-20020727/include/entry.h
--- cdk-4.9.10-20020727.orig/include/entry.h	Sat Jul 27 18:34:02 2002
+++ cdk-4.9.10-20020727/include/entry.h	Fri Mar 28 14:44:05 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/fscale.h cdk-4.9.10-20020727/include/fscale.h
--- cdk-4.9.10-20020727.orig/include/fscale.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/fscale.h	Fri Mar 28 14:44:13 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/fselect.h cdk-4.9.10-20020727/include/fselect.h
--- cdk-4.9.10-20020727.orig/include/fselect.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/fselect.h	Fri Mar 28 14:44:22 2003
@@ -12,13 +12,13 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
-#include <entry.h>
-#include <scroll.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
+#include <cdk/entry.h>
+#include <cdk/scroll.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/graph.h cdk-4.9.10-20020727/include/graph.h
--- cdk-4.9.10-20020727.orig/include/graph.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/graph.h	Fri Mar 28 14:44:31 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/histogram.h cdk-4.9.10-20020727/include/histogram.h
--- cdk-4.9.10-20020727.orig/include/histogram.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/histogram.h	Fri Mar 28 14:44:41 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/itemlist.h cdk-4.9.10-20020727/include/itemlist.h
--- cdk-4.9.10-20020727.orig/include/itemlist.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/itemlist.h	Fri Mar 28 14:44:49 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/label.h cdk-4.9.10-20020727/include/label.h
--- cdk-4.9.10-20020727.orig/include/label.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/label.h	Fri Mar 28 14:44:58 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/marquee.h cdk-4.9.10-20020727/include/marquee.h
--- cdk-4.9.10-20020727.orig/include/marquee.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/marquee.h	Fri Mar 28 14:45:06 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/matrix.h cdk-4.9.10-20020727/include/matrix.h
--- cdk-4.9.10-20020727.orig/include/matrix.h	Sat Jul 27 18:36:19 2002
+++ cdk-4.9.10-20020727/include/matrix.h	Fri Mar 28 14:45:14 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/mentry.h cdk-4.9.10-20020727/include/mentry.h
--- cdk-4.9.10-20020727.orig/include/mentry.h	Sat Jul 27 18:37:27 2002
+++ cdk-4.9.10-20020727/include/mentry.h	Fri Mar 28 14:45:22 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/menu.h cdk-4.9.10-20020727/include/menu.h
--- cdk-4.9.10-20020727.orig/include/menu.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/menu.h	Fri Mar 28 14:45:31 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/radio.h cdk-4.9.10-20020727/include/radio.h
--- cdk-4.9.10-20020727.orig/include/radio.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/radio.h	Fri Mar 28 14:45:45 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/scale.h cdk-4.9.10-20020727/include/scale.h
--- cdk-4.9.10-20020727.orig/include/scale.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/scale.h	Fri Mar 28 14:45:57 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/scroll.h cdk-4.9.10-20020727/include/scroll.h
--- cdk-4.9.10-20020727.orig/include/scroll.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/scroll.h	Fri Mar 28 14:46:07 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/selection.h cdk-4.9.10-20020727/include/selection.h
--- cdk-4.9.10-20020727.orig/include/selection.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/selection.h	Fri Mar 28 14:46:17 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/slider.h cdk-4.9.10-20020727/include/slider.h
--- cdk-4.9.10-20020727.orig/include/slider.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/slider.h	Fri Mar 28 14:46:26 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/swindow.h cdk-4.9.10-20020727/include/swindow.h
--- cdk-4.9.10-20020727.orig/include/swindow.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/swindow.h	Fri Mar 28 14:46:34 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/template.h cdk-4.9.10-20020727/include/template.h
--- cdk-4.9.10-20020727.orig/include/template.h	Sat Jul 27 18:39:09 2002
+++ cdk-4.9.10-20020727/include/template.h	Fri Mar 28 14:46:42 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/traverse.h cdk-4.9.10-20020727/include/traverse.h
--- cdk-4.9.10-20020727.orig/include/traverse.h	Wed Jul 10 23:41:50 2002
+++ cdk-4.9.10-20020727/include/traverse.h	Fri Mar 28 14:46:51 2003
@@ -6,15 +6,13 @@
 #ifndef CDKTRAVERSE_H
 #define CDKTRAVERSE_H 1
 
-#include "cdk.h"
-
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
diff -Nur cdk-4.9.10-20020727.orig/include/viewer.h cdk-4.9.10-20020727/include/viewer.h
--- cdk-4.9.10-20020727.orig/include/viewer.h	Sun Jul 14 23:49:37 2002
+++ cdk-4.9.10-20020727/include/viewer.h	Fri Mar 28 14:47:00 2003
@@ -12,11 +12,11 @@
 
 #ifndef CDK_H
 #define CDKINCLUDES
-#include <cdk.h>
+#include <cdk/cdk.h>
 #undef CDKINCLUDES
-#include <binding.h>
-#include <cdkscreen.h>
-#include <cdk_objs.h>
+#include <cdk/binding.h>
+#include <cdk/cdkscreen.h>
+#include <cdk/cdk_objs.h>
 #endif
 
 /*
