--- cdk-5.0-20161210/Makefile.in.orig	2015-09-29 01:37:35.000000000 +0200
+++ cdk-5.0-20161210/Makefile.in	2017-07-19 05:52:26.274373712 +0200
@@ -296,7 +296,7 @@
 all cdklib :: $(CDKLIB)
 
 @MAKE_NORMAL@cdkshlib $(OLD_SHLIB_FILE) :: $(CDKSHOBJS)
-@MAKE_NORMAL@	gcc -shared -Wl,-soname=$(OLD_SONAME) $(LDFLAGS) $(LIBS) -o $(OLD_SHLIB_FILE) $(CDKSHOBJS)
+@MAKE_NORMAL@	gcc -shared -Wl,-soname=$(OLD_SONAME) $(LDFLAGS) $(LIBS) -o $(OLD_SHLIB_FILE) $(CDKSHOBJS) -lncurses
 
 #
 # Make the examples directory.
